druid-shell/src/platform/windows/accels.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Wrappers for Windows of Accelerate Table.
+
+use winapi::ctypes::c_int;
+use winapi::shared::windef::*;
+use winapi::um::winuser::*;
+
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+struct WindowHandle(HWND);
+unsafe impl Send for WindowHandle {}
+
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+struct AccelHandle(HACCEL);
+unsafe impl Send for AccelHandle {}
+unsafe impl Sync for AccelHandle {}
```

Comment:
Do we know that these pointers are thread safe? Can we include a link to some reference or something?

Reply:
Yeah, if I understand correctly, all handles you obtain from functions in Kernel32 are thread-safe, unless the MSDN Library article for the function explicitly mentions it is not.

See https://docs.microsoft.com/en-us/windows/win32/wsw/thread-safety

And I added a comment for that.